docs/NetCore_GettingStarted.md

```diff
@@ -25,29 +25,34 @@
 	- Do not separate analyzer tests from code fix tests. If the analyzer has a code fix, then write all your tests as code fix tests.
 		- Calling `VerifyCodeFixAsync(source, source)` verifies that the analyzer either does not produce diagnostics, or produces diagnostics where no code fix is offered.
 		- Calling `VerifyCodeFixAsync(source, fixedSource)` verifies the diagnostics (analyzer testing) and verifies that the code fix on source produces the expected output.
-- Run the analyzer locally against `dotnet/runtime` [instructions](#Testing-against-the-Runtime-repo). 
-	- Use the failures to discover nuance and guide the implementation details. 
-	- Run the analyzer against `dotnet/roslyn` [instruction](#Testing-against-the-Roslyn-repo), and with `dotnet/aspnetcore` if feasable. 
-	- Review each of the failures in those repositories and determine the course of action for each. 
+- Run the analyzer locally against `dotnet/runtime` and `dotnet/roslyn-analyzers` [instructions](#Testing-against-the-Runtime-and-Roslyn-analyzers-repo).
+	- Review each of the failures in those repositories and determine the course of action for each.
+	- Use the failures to discover nuance and guide the implementation details.
+	- Run the analyzer against `dotnet/roslyn` [instruction](#Testing-against-the-Roslyn-repo), and if feasable with `dotnet/aspnetcore` repos.
+	- Document for review: matching and non-matching scenarios, including any discovered nuance.
+	- Failures in all `dotnet` repos are addressed.
 - Document for review: severity, default, categorization, numbering, titles, messages, and descriptions.
-- Document for review: matching and non-matching scenarios, including any discovered nuance. 
 - Create the appropriate documentation for [docs.microsoft.com](https://github.com/MicrosoftDocs/visualstudio-docs-pr/tree/master/docs/code-quality) within **ONE WEEK**, instructions available on OneNote. External contributors should create an issue at https://github.com/microsoftDocs/visualstudio-docs/issues with a subject `Add documentation for analyzer rule [Your Rule ID]`. 
 - PR merged into `dotnet/roslyn-analyzers`. 
-- Failures in `dotnet/runtime` addressed. 
 
-## Testing against the Runtime repo 
+## Testing against the Runtime and Roslyn-analyzers repo 
```

Comment:
Tagging @vatsalyaagrawal - as we discussed yesterday, it would be good if we author an integration test leg for the repo that executes such real world tests directly on PR. It would serve multiple purposes:

1. [P0 - reliability] Integration test will fail in presence of any AD0001/analyzer exception
2. [P1 - reliability] Integration test will apply code fixes and/or fix all and fail in presence of exceptions
3. [P1 - perf] Integration test with execute with [reportanalyzer](https://github.com/dotnet/roslyn/blob/master/docs/analyzers/Report%20Analyzer%20Format.md) switch so perf data can be logged and viewed by developers. We can even consider failing the test in presence of an outlier expensive analyzer based on some threshold.
4. [P2 - enhancements] Test can be executed locally to create an automated branch with a commit that applies a FixAll application for a specific code action registered by a fixer. Developers can submit a PR from this branch.

I am going to file a tracking issue for this work item and prioritize working on it this sprint.

Reply:
> it would be good if we author an integration test leg for the repo that executes such real world tests directly on PR.

That is a great idea @mavasani, i assume we still keep it in the doc, at least until that work gets done